packages/test-runner-intercept/browser/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Intercept a module in order to change its implementation.
+ * @param {string} moduleName The name of the module to intercept. When intercepting the import of a module, the name should exactly match the name of the import.
+ * @returns {Promise<Record<string, func> | void >} Writable object with every named export of the intercepted module as a property.
+ * If the module to be intercepted contains a default export, the export will be available in the `default` property. Only function expressions are supported when intercepting default exports.
+ */
+export async function interceptModule(moduleName) {
+  let module;
+  try {
+    module = await import(`/__intercept__/${moduleName}`);
+  } catch (e) {
+    throw new Error(
+      `Module interception is not active. Make sure the \`interceptModulePlugin\` of \`@web/test-runner-intercept\` is added to the Test Runner config.`,
+    );
+  }
+  if (module.__wtr_error__) {
+    throw new Error(module.__wtr_error__);
+  }
+  return module.__wtr_intercepted_module__;
+}
```

Comment:
if this is a TS package, I think it makes sense to use TS here too as we do in other similar cases?

Reply:
This code is browser code. I took over the convention of `@web/test-runner-commands` to only have the server-side code in TS. The script of @web as well don't take into account a mix of browser and server-sider code in TS.

Reply:
Yup, indeed. :-)

But not sure "want" is the correct term here. ;-) I just reused the same pattern to not get into trouble.